.github/workflows/ci.yml

```diff
@@ -12,7 +11,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest]
-        python-version: ['3.9', '3.10', '3.11']
+        python-version: ["3.10", "3.11"]
```

Comment:
Awesome! Thanks for getting to this so quickly!

Reply:
No problem, happy to make these changes :D